README.md

```diff
@@ -337,5 +326,87 @@ Contact the dev team for the following:
 
 Once you have access to AWS LZ2 platform, login to the any of the environments to get access to the following: 
```

Comment:
remove "the" and add "login to any of the environments using your IDIR credentials to get access to the following:"